DuckDuckGo/IconSettingsViewController.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  IconSettingsViewController.swift
+//  DuckDuckGo
+//
+//  Copyright © 2020 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import Core
+
+class IconSettingsViewController: UICollectionViewController {
+
+    private var icons = Icon.allCases
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        applyTheme(ThemeManager.shared.currentTheme)
+        initSelection()
```

Comment:
If you're going to add `initSelection()` call to `decorate(with:)` (see below) then it should be fine to remove this one. :)

Reply:
✅